source/extensions/filters/udp/dns_filter/dns_filter.h

```diff
@@ -51,25 +63,78 @@ class DnsFilterEnvoyConfig {
   }
 
   Stats::Scope& root_scope_;
+
   mutable DnsFilterStats stats_;
-  mutable DnsVirtualDomainConfig virtual_domains_;
+  DnsVirtualDomainConfig virtual_domains_;
+  std::vector<Matchers::StringMatcherPtr> known_suffixes_;
+  absl::flat_hash_map<std::string, uint64_t> domain_ttl_;
+  bool forward_queries_;
+  AddressConstPtrVec resolvers_;
+  std::chrono::milliseconds resolver_timeout_ms_;
```

Comment:
nit: ms_ is redundant now that you are using chrono types. Same elsewhere if I missed anything.

Reply:
Fixed